sql/sql_plugin.cc

```diff
@@ -3237,8 +3240,9 @@ static int construct_options(MEM_ROOT *mem_root, st_plugin_int *tmp,
       (char *)mem_root->Alloc(plugin_name_len + plugin_dash.length + 1);
   strxmov(plugin_name_with_prefix_ptr, plugin_dash.str, plugin_name_ptr, NullS);
 
-  if (tmp->load_option != PLUGIN_FORCE &&
-      tmp->load_option != PLUGIN_FORCE_PLUS_PERMANENT) {
+  if ((tmp->load_option != PLUGIN_FORCE &&
+       tmp->load_option != PLUGIN_FORCE_PLUS_PERMANENT) ||
+      (strcmp(plugin_name, "ROCKSDB") == 0)) {
```

Comment:
This is needed when I run setup command in dev server to build a database. Without this change, the setup script will fail.

Reply:
What is the command that the setup script executes and what error is it getting?

Reply:
It's about handling the command options. When executing mysqld command in the setup script, we pass --rocksdb as a command parameter. After changing Rocksdb as mandatory, this option is considered loose anymore. And will throw an error about unknown option and fail the mysqld. The change here helps to add the option to option list to pass the option check.

Reply:
I don't like the special casing here, but what has to be done has to be done. Perhaps the scripts should no longer pass `--rocksdb` (as they are not passing `--innodb` currently, right?), and then later this could be removed.

Reply:
@sunshine-Chun , if we revert this change, and if MTR passes, we can upgrade script to remove that --rocksdb instead.

Reply:
Sounds good. Let me remove this line and see if MTR test passes

Reply:
Looks like the MTR passed. Will remove this line.